cuda_core/docs/source/release/0.5.x-notes.rst

```diff
@@ -0,0 +1,39 @@
+.. SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. currentmodule:: cuda.core.experimental
+
+``cuda.core`` 0.5.x Release Notes
+=================================
+
+
+Highlights
+----------
+
+None.
+
+
+Breaking Changes
+----------------
+
+- Python ``bool`` objects are now converted to C++ ``bool`` type when passed as kernel
+  arguments.  Previously, they were converted to ``int``.  This brings them inline
+  with ``ctypes.c_bool`` and ``numpy.bool_``.
```

Comment:
Q: I think this no longer applies? Should we just add another entry to mention the perf improvement

Reply:
Ahhhh ok. Let us document this as a bug fix instead of breaking change. It was not designed to be casting a Python bool to c++ int32. We can wait until the CI finishes, change the rel notes, and admin-merge.